controllers/kafkauser_controller.go

```diff
@@ -154,6 +154,7 @@ type KafkaUserReconciler struct {
 
 // +kubebuilder:rbac:groups=kafka.banzaicloud.io,resources=kafkausers,verbs=get;list;watch;create;update;patch;delete;deletecollection
 // +kubebuilder:rbac:groups=kafka.banzaicloud.io,resources=kafkausers/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=kafka.banzaicloud.io,resources=kafkausers/finalizer,verbs=create;update;patch;delete
```

Comment:
```suggestion
// +kubebuilder:rbac:groups=kafka.banzaicloud.io,resources=kafkausers/finalizers,verbs=create;update;patch;delete
```

Reply:
Fixed in https://github.com/banzaicloud/koperator/compare/50b41f0bbe5458899769a98dc20153c011f7f752..1df5df1d3d9248a0dff0b6dc4cb311acf210045a.